.github/workflows/release.yml

```diff
@@ -7,27 +7,119 @@ on:
         description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
         default: '7.y.z'
+
+env:
+  IMAGE: quay.io/eclipse/che-dashboard
+
```

Comment:
are you missing a cache image here? 

https://github.com/eclipse/che-dashboard/blob/master/.github/workflows/ci-multiarch.yaml#L21

Reply:
the cache is under discussion in https://github.com/eclipse/che-dashboard/pull/119#discussion_r590943118